src/components/tabs.tsx

```diff
@@ -168,12 +178,17 @@ export const Tabs = ({ children, active, className, ...props }: TabsProps) => {
                 className={cn(
                   'tabs__header__item',
                   {
+                    'tabs__header__item--bordered': bordered,
+                    'tabs__header__item--disabled': disabled,
                     'tabs__header__item--active': id === activeFromPropsOrState,
                   },
                   className
                 )}
+                aria-disabled={disabled ? true : undefined}
+                onFocus={disabled ? (event) => event.target.blur() : undefined}
```

Comment:
I'm not sure that this is needed, because of the tabIndex -1 below

Reply:
I tried locally with and without this line and didn't see any difference, but maybe it was done for a reason I'm missing?